internal.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Internal components of the Pack SDK. You should not rely on these, since these can
+ * change without warning.
+ *
+ * @module internal
+ * @hidden
+ */
+
+export type {CompiledAutocompleteSnippet} from './documentation/types';
+export type {CompiledExample} from './documentation/types';
+export type {CompiledExampleSnippet} from './documentation/types';
+export type {PackUpload} from './compiled_types';
+
+export type {MetadataFormulaMetadata} from './api';
+export {legacyMarshalValue} from './helpers/legacy_marshal';
+export {legacyUnmarshalValue} from './helpers/legacy_marshal';
+export {marshalValueToString} from './runtime/common/marshaling';
+export {marshalError} from './runtime/common/marshaling';
+export {unmarshalError} from './runtime/common/marshaling';
+export {unmarshalValueFromString} from './runtime/common/marshaling';
+
+// Schema types
+export type {UnionType} from './api_types';
+export {isArray} from './schema';
+export {isObject} from './schema';
+export {isArrayType} from './api_types';
+
+// Schema manipulation / transformation
+export {paramDefHelper} from './helpers/migration';
+export {postSetupMetadataHelper} from './helpers/migration';
+export {objectSchemaHelper} from './helpers/migration';
+
+export {normalizeSchema} from './schema';
+export {normalizeSchemaKey} from './schema';
+export {normalizePropertyValuePathIntoSchemaPath} from './schema';
+
+// Fetcher helpers
+export {transformBody} from './handler_templates';
+
+
+export * as legacyImports from './legacy_exports';
```

Comment:
How bad is the refactoring in the `packs` repo? Hopefully it will work there to just change the imports and not have to destructure all the legacy bits?